cmds/exp/watch/watch.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2015-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"fmt"
+	"flag"
+	"os"
+	"os/exec"
+	"time"
+	"bufio"
+
+)
+
+var (
+	t       	= flag.Bool("t", false, "Don't print header")
+	n       	= flag.Int64("n", 2, "Loop period in SEC, default 2")
```

Comment:
Use `flag.Duration`

Reply:
Flag.Duration by defaults assumes the time to be in nano seconds. So a CLI argument in nanosec would be too uncomfortable